### DISCUSSION TITLE
How active is the project?

### DISCUSSION THREAD

**Discussion by Croworbit** [lightspark/lightspark#1113] [General]
hello, how active is the project? anything big planned? what can I expect of this software's compatability in practice?

**Comment by eiglow** [comment]
My perspective is mainly with running flash games, and for this, its generally pretty good. Some games might crash or run poorly, but many games are working okay.

I think its a decently active project, there are 2 devs mainly working on it right now, and a couple of others (including me) do some QA/testing work sometimes. Its certainly not as active as Ruffle, but usually whenever I make a bug report, as long as I explained the bug properly, its fixed quickly.
